loopout.go

```diff
@@ -719,11 +719,23 @@ func (s *loopOutSwap) payInvoiceAsync(ctx context.Context,
 	paymentSuccess := err == nil &&
 		paymentStatus.State == lnrpc.Payment_SUCCEEDED
 
-	if err := s.swapKit.server.ReportRoutingResult(
-		ctx, s.swapInfo().SwapHash, s.swapInvoicePaymentAddr, pluginType,
-		paymentSuccess, int32(attempts), dt.Milliseconds(),
-	); err != nil {
-		s.log.Warnf("Failed to report routing result: %v", err)
+	// RoutingPluginNone is requested for prepays which we don't want to
+	// report.
+	if pluginType != RoutingPluginNone {
```

Comment:
Is there any point in reporting swap payments that use `RoutingPluginNone` so that we can compare them to ones that use the plugin? I'd guess not because they're also affected by the mission control changes? 

If yes, I think we should pass in a `reportPlugin` bool ?

Reply:
The passed in bool is solves this nicely.